src/mixins/tipUrls.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import { BACKEND_URL } from '../popup/utils/constants';
+import { getTwitterAccountUrl } from '../popup/utils/helper';
+
+export default {
+  data: () => ({
+    verifiedUrls: [],
+    blacklistedUrls: [],
+  }),
+  async created() {
+    try {
+      this.verifiedUrls = (await axios.get(`${BACKEND_URL}/verified`)).data;
+      this.blacklistedUrls = (await axios.get(`${BACKEND_URL}/static/wallet/graylist`)).data;
+    } catch (e) {
+      this.$logError({ e, action: 'fetch-verified' });
+    }
```

Comment:
```suggestion
    const [{ data: verified }, { data: graylist }] = await Promise.all([
      axios.get(`${BACKEND_URL}/verified`),
      axios.get(`${BACKEND_URL}/static/wallet/graylist`),
    ]);
    this.verifiedUrls = verified;
    this.blacklistedUrls = graylist;
```
I propose to:
- make requests simultaneously
- drop specific error logging because anyway it should be caught by global handlers